qnx_qemu/README.md

```diff
@@ -4,6 +4,226 @@
+```
+
+after that
+1-open web browser, go to <www.qnx.com>
+2-Select the Developers tab, then select QNX Software Center.
+3-Scroll down and click the link labeled QNX Software Center version_number build_number - Linux Hosts or QNX Software Center version_number build_number - macOS Hosts.
+4-Right-click Download Now and copy the link address to your clipboard.
+5-Paste the link address into the curl command described in the next step.
+
+```bash
+curl -v -L --cookie ~/.qnx/myqnxcookies.auth installer_URL  > qnx-setup-lin.run
+```
+
+6- To ensure you are using the official installer, verify that the checksum of the downloaded file matches the checksum posted on the QNX website.
+
+```bash
+sha256sum qnx-setup-lin.run
+#Compare against the checksum shown on the QNX download page.
+```
+
+7- Make the installer executable.
+
+```bash
+chmod a+x ./qnx-setup-lin.run
+```
+
+8- For Linux, enter the following command, which installs the QNX Software Center in ~/qnxinstall/qnxsoftwarecenter/.
+
+```bash
+~/qnx-setup-lin.run force-override disable-auto-start agree-to-license-terms ~/qnxinstall
+```
+
+- **Download QNX SDP 8.0**
+
+From **QNX Software Center - Linux Hosts** we previously installed , install **QNX Software Development Platform 8.0**.
+
+OR you can do it using CLI
+
+```bash
+cd ~/qnxinstall/qnxsoftwarecenter
+
+./qnxsoftwarecenter_clt -syncLicenseKeys -myqnx.user="$SCORE_QNX_USER" -myqnx.password="$SCORE_QNX_PASSWORD" -addLicenseKey license_key -listLicenseKeys
+
+./qnxsoftwarecenter_clt -mirrorBaseline="qnx800" -myqnx.user="$SCORE_QNX_USER" -myqnx.password="$SCORE_QNX_PASSWORD"
+
+./qnxsoftwarecenter_clt -cleanInstall -destination ~/qnx800 -installBaseline com.qnx.qnx800 -myqnx.user="$SCORE_QNX_USER" -myqnx.password="$SCORE_QNX_PASSWORD"
+
+cd - > /dev/null
+```
+
+This contains the toolchains and `license` folder required for activation.
+check this QNX link for more info <https://www.qnx.com/developers/docs/qsc/com.qnx.doc.qsc.inst_larg_org/topic/create_headless_installation_linux_macos.html>
+
+- **Source SDP environment**
+
+```bash
+source ~/qnx800/qnxsdp-env.sh 
+#The commands below confirm the environment was sourced successfully.
+echo "$QNX_HOST" "$QNX_TARGET"
+```
+
+- **Register QNX license**
+
+```bash
+sudo mkdir -p /opt/score_qnx/license
+sudo cp ~/.qnx/license/licenses /opt/score_qnx/license/licenses
+sudo chmod 644 /opt/score_qnx/license/licenses
+```
+
+OR we can use symbolic linking
+
+```bash
+sudo install -d -m 755 /opt/score_qnx/license
+LIC_SRC="$(readlink -f "$HOME/.qnx/license/licenses")"
+sudo ln -sfn "$LIC_SRC" /opt/score_qnx/license/licenses
+chmod 644 "$LIC_SRC"
+```
+
+- **KVM acceleration**
+
+QEMU can use hardware virtualization via **KVM** for near-native speed.
+
+- If your host supports KVM, keep `--enable-kvm` in `reference_integration/qnx_qemu/scripts/run_qemu.sh`.
```

Comment:
Please make it parametrizable in a script, instead of requesting to modify scripts inside repo.